packages/test/test-end-to-end-tests/src/test/data-virtualization/snapshotTestCache.ts

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ISnapshot } from "@fluidframework/driver-definitions/internal";
+import {
+	getKeyForCacheEntry,
+	type ICacheEntry,
+	type IFileEntry,
+	type IPersistedCache,
+} from "@fluidframework/odsp-driver-definitions/internal";
+
+export interface ValueWithSnapshot {
+	value: ISnapshot;
+}
+
+export class SnapshotCache implements IPersistedCache {
```

Comment:
nit: Should we rename the class to "TestSnapshotCache"?

Reply:
Great Idea